shared/react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -38,7 +38,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         logFile = this.getFileStreamPath("android.log");
```

Comment:
the up arrow stuff is just leftover junk that i assume someone might use. i didn't touch it much. the real flow of doing a log send from RN is through https://github.com/keybase/client/blob/2f7894a161b82ca39138826981e9c0035da2632a/shared/react-native/android/app/src/main/java/io/keybase/ossifrage/modules/LogSend.java